tests/modules/programs/delta/delta-final-package.nix

```diff
@@ -0,0 +1,35 @@
+{ config, ... }:
+
+{
+  programs.delta = {
+    enable = true;
+    package = config.lib.test.mkStubPackage {
+      name = "delta";
+      buildScript = ''
+        mkdir -p $out/bin
+        touch $out/bin/delta
+        chmod 755 $out/bin/delta
+      '';
+    };
+    options = {
+      features = "line-numbers decorations";
+      syntax-theme = "Dracula";
+      decorations = {
+        commit-decoration-style = "bold yellow box ul";
+        file-style = "bold yellow ul";
+        file-decoration-style = "none";
+      };
+    };
+    enableGitIntegration = true;
+  };
+  programs.git.enable = false;
```

Comment:
Alternatively, we could assert that this doesn't happen, but that seems overly restrictive.